share/pot/init.sh

```diff
@@ -84,6 +84,11 @@ pot-init()
 	mkdir -p /usr/local/etc/newsyslog.conf.d
 	mkdir -p /var/log/pot
 
+	if ! _is_pot_tmp_dir ; then
+		_error "The POT_TMP directory has not been created - aborting"
```

Comment:
Wouldn't it make sense to set POT_TMP to /tmp globally in case it isn't set instead of repeating `${POT_TMP:-/tmp}` all over the code?

Reply:
It is. In `pot.default.conf` you have default values defined.
This file **should** not be touched and is automatically overwritten by `pkg`. If touched, `pkg` won't update it.
Repeating `${POT_TMP:-/tmp}` is a way to be extra sure that the variable is always resolved.

Reply:
Ok